plugin/storage/cassandra/options.go

```diff
@@ -214,6 +214,10 @@ func addFlags(flagSet *flag.FlagSet, nsConfig namespaceConfig) {
 		nsConfig.namespace+suffixPassword,
 		nsConfig.Authenticator.Basic.Password,
 		"Password for password authentication for Cassandra")
+	flagSet.String(
+		nsConfig.namespace+suffixAuth,
+		"",
+		"The comma-separated list of Allowed password authentication for Cassandra.")
```

Comment:
```suggestion
		"The comma-separated list of Allowed password authenticators for Cassandra.")
```
add a link to the list of acceptable values